pages/tic-tac-toe/index.tsx

```diff
@@ -113,7 +113,7 @@ export default function T() {
             );
             const randomChoice = choicesMinusMine[randomIndex];
             const bestChoice = findMoveForRobot(allMyChoices, oIndexes);
-            const choice = bestChoice || randomChoice;
+            const choice = bestChoice ?? randomChoice;
```

Comment:
This handles a bug where if the best choice is `0` the best choice wasn't being chosen (since it is falsy)